.github/workflows/main.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Run Integration Tests
         run: |
           cd it-selenium
-          mvn -V -ntp -Dwebdriver.gecko.driver=$GECKOWEBDRIVER/geckodriver install
+          mvn -V -ntp install
```

Comment:
I think the newer version of the firefox selenium plugin is managing the geckodriver download automatically

Reply:
Nice